Bio/KEGG/KGML/KGML_parser.py

```diff
@@ -80,7 +80,109 @@ def parse(handle, debug=0):
 
 
 class KGMLParser(object):
-    """Parses a KGML XML Pathway entry into a Pathway object."""
+    """Parses a KGML XML Pathway entry into a Pathway object.
+        Example: Read and parse large metabolism file
```

Comment:
Strange indentation and missing blank line, see PEP257 https://www.python.org/dev/peps/pep-0257/